### PR TITLE
Add "julia" check

### PR DESCRIPTION
**Pull request by ericphanson**
Fixes https://github.com/JuliaRegistries/RegistryCI.jl/issues/112 and addresses @StefanKarpinski's comment https://github.com/JuliaRegistries/RegistryCI.jl/issues/285#issuecomment-718930717

### CODE REVIEW

**Comment by ericphanson** [comment]
bors try

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "RegistryCI"
 uuid = "0c95cc5f-2f7e-43fe-82dd-79dbcba86b32"
 authors = ["Dilum Aluthge <dilum@aluthge.com>", "Fredrik Ekre <ekrefredrik@gmail.com>"]
-version = "4.1.1"
+version = "4.2.1"
```
Comment (ericphanson) [inline Project.toml:4]:
```suggestion
version = "4.2.0"
```

**Review by DilumAluthge — APPROVED** [review]

**Comment by ericphanson** [comment]
bors try

**Comment by ericphanson** [comment]
bors merge

**Pull request merged**